test/uk/gov/hmrc/selfassessmentassist/v1/connectors/DefaultRdsAuthConnectorSpec.scala

```diff
@@ -45,11 +45,10 @@ class DefaultRdsAuthConnectorSpec
     with Injecting
     with MockAppConfig
     with EitherValues {
-  var port: Int = _
-
+  val httpClient: HttpClient              = app.injector.instanceOf[HttpClient]
   private val actorSystem: ActorSystem    = actor.ActorSystem("unit-testing")
   implicit val materializer: Materializer = Materializer.matFromSystem(actorSystem)
-  val httpClient: HttpClient              = app.injector.instanceOf[HttpClient]
+  var port: Int                           = _
```

Comment:
Can we replace the `var` with a `def` (and remove the setting of the port in `beforeAll`):
```suggestion
  def port: Int              = wireMockServer.port()
```

Reply:
Similarly in IfsConnectorSpec RdsConnectorSpec, NrsConnectorSpec and InsightConnectorSpec